.github/workflows/test_helper.yml

```diff
@@ -58,10 +40,58 @@ jobs:
           sqlfluff lint products/knownprojects/sql --templater=jinja
           sqlfluff lint products/pluto/pluto_build/sql/
           
-
-          # dbt requires slightly special logic
-          ./bash/sqlfluff_dbt.sh lint green_fast_track
-          ./bash/sqlfluff_dbt.sh lint zoningtaxlots
+  dbt_tests:
+    name: dbt tests
+    runs-on: ubuntu-22.04
+    container:
+      image: nycplanning/dev:${{ inputs.image_tag }}
+    defaults:
+      run:
+        shell: bash
+        working-directory: products/${{ matrix.project }}
+    env:
+      BUILD_ENGINE_HOST: postgis
+      BUILD_ENGINE_USER: postgres
+      BUILD_ENGINE_PASSWORD: postgres
+      BUILD_ENGINE_PORT: "5432"
+      BUILD_ENGINE_DB: postgres
+      BUILD_ENGINE_SCHEMA: ${{ matrix.project }}
+      # sqlfluff env vars
+      VERSION: "2024-05-01"
+      VERSION_PREV: "2024-04-01"
+    services:
+      postgis:
+        image: postgis/postgis:13-3.3-alpine
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+          --shm-size=2gb
+        ports:
+          - 5432:5432
+    strategy:
+      fail-fast: false
+      matrix:
+        project:
+          - green_fast_track
+          - zoningtaxlots
+    steps:
+      - uses: actions/checkout@v4
+      - name: setup
+        run: |
+          git config --global --add safe.directory /__w/data-engineering/data-engineering
```

Comment:
is this needed for `pre-commit` to work

if it doesn't add much time, could probably be `./bash/docker_container_setup.sh` like the pytest section in this file

Reply:
Yeah, but that also sets up mc connection to DO spaces, which I'd prefer to not do in this step. I guess I could tweak that command to default to empty strings rather

Reply:
mc command is not happy with it

https://github.com/NYCPlanning/data-engineering/actions/runs/9875059942/job/27270890880#step:4:14

I'd rather leave this git command than supply a bunch of env vars that aren't needed otherwise